kubetest/kops.go

```diff
@@ -487,3 +530,89 @@ func (k *kops) runKopsDump() (*kopsDump, error) {
 
 	return dump, nil
 }
+
+// getRandomAWSZones looks up all regions, and the availability zones for those regions.  A random
+// region is then chosen and the AZ's for that region is returned.  If HA is required a regions with 3
+// or more zones is returned.
+func getRandomAWSZones(masterCount int) (*[]string, error) {
```

Comment:
I was unclear before - I suggest this should return `([]string, error)`, because []string is already a ref type, so it's already nullable.